app/models/mixins/per_ems_worker_mixin.rb

```diff
@@ -18,7 +18,7 @@ def all_ems_in_zone
     end
 
     def all_valid_ems_in_zone
-      all_ems_in_zone.select(&:authentication_status_ok?)
+      all_ems_in_zone.where(:enabled => true).select(&:authentication_status_ok?)
```

Comment:
We tend to use class member variables instead.
Was told by matthew/fry that in rails 5 they are the same thing.
But could have misunderstood and did not verify.
(Everything you read on the internet is true. no?)